Makefile.am

```diff
@@ -34,3 +34,10 @@ tests/run-test-suite:
 	-git branch --track run-test-suite origin/run-test-suite
 	-git worktree prune
 	git worktree add $@ run-test-suite
+
+packaging:
+	git branch --track packaging origin/packaging
```

Comment:
`-git branch --track $@ origin/$@`

The leading `-` will keep it from stopping if the command fails because it is already done.